src/stac_utils/http.py

```diff
@@ -174,21 +215,39 @@ def call_api(
         return data
 
     def get(self, *args, **kwargs):
+        """
+        Convenience wrapper for GET
+        """
         return self.call_api("GET", *args, **kwargs)
 
     def post(self, *args, **kwargs):
+        """
+        Convenience wrapper for POST
```

Comment:
might be helpful for these have an example, eg you'd use `self.van.post("signups", body=payload)` or `self.van.get(f"events/{event_id}?$expand=locations,roles,shifts")`